robottelo/ui/base.py

```diff
@@ -194,7 +194,11 @@ def delete(self, name, really=True, dropdown_present=False,
                search_query=None):
         """Delete an added entity, handles both with and without dropdown."""
         self.logger.debug(u'Deleting entity %s', name)
-        searched = self.search(name, _raw_query=search_query)
+        # Overridden search methods do not support search queries
```

Comment:
For example. some pages do not have search field and button hence do not support search queries. 
For example, overridden search method from `ui.Trend`: https://github.com/SatelliteQE/robottelo/blob/master/robottelo/ui/trend.py#L45
There is no way to pass search query.